src/dispatch.jl

```diff
@@ -150,3 +150,75 @@ function plot_electric_dispatch(d::Dict; title="Electric Systems Dispatch", save
     PlotlyJS.plot(traces, layout)  # will not produce plot in a loop
 
 end
+
+function plot_thermal_dispatch(d::Dict; title="Thermal Systems Dispatch", save_html=false)
+
+    traces = PlotlyJS.GenericTrace[]
+
+    layout = PlotlyJS.Layout(
+        title_text = "",
+        xaxis_title_text = "time step",
+        yaxis_title_text = "kW",
+        yaxis2 = (
+            title = "temp [C]",
+            overlaying = "y",
+            side = "right"
+        )
+    )
+
+    if "ExistingChiller" in keys(d)
+        T = length(d["ExistingChiller"]["year_one_electric_consumption_series"])
+    else
+        T = 8760
+    end
+    
+    # x axis resolution is determined by length of T.
+    x_axis = DateTime(2021):Dates.Minute(Int(60*(8760/T))):DateTime(2021,12,31,23,45)
```

Comment:
@rathod-b I like that the thermal dispatch plot has a date axis. Will you please add this to the electric dispatch? Also, we should not make the year hard-coded. Please make year an input to both dispatch functions, defaulting to 2021.

Reply:
Updated in [7e4301b](https://github.com/NREL/REoptPlots.jl/pull/1/commits/7e4301b825ce1f21328b023b3ba1262f3ac096b8)